richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/richtext/RichText.java

```diff
@@ -130,8 +130,8 @@ public void start(Stage primaryStage) {
         textColorPicker.valueProperty().addListener((o, old, color) -> updateTextColor(color));
         backgroundColorPicker.valueProperty().addListener((o, old, color) -> updateBackgroundColor(color));
 
-        undoBtn.disableProperty().bind(Bindings.not(area.undoAvailableProperty()));
-        redoBtn.disableProperty().bind(Bindings.not(area.redoAvailableProperty()));
+        area.undoAvailableProperty().values().feedTo(undoBtn.disableProperty());
+        area.redoAvailableProperty().values().feedTo(redoBtn.disableProperty());
```

Comment:
I'm not modeling good coding by storing the resulting `Subscription`, but... is that really necessary?